react/features/transcribing/components/TranscribingLabel.web.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { translate } from '../../base/i18n/index';
+
+import { CircularLabel } from '../../base/label/index';
+import Tooltip from '@atlaskit/tooltip';
+
+/**
+ * The type of the React {@code Component} props of {@link TranscribingLabel}.
+ */
+type Props = {
+
+    /**
+     * Invoked to obtain translated strings.
+     */
+    t: Function,
+
+    /**
+     * Boolean value indicating current transcribing status
+     */
+    _transcribing: boolean
+};
+
+/**
+ * React Component for displaying a label when a transcriber is in the
+ * conference.
+ *
+ * @extends Component
+ */
+class TranscribingLabel extends Component<Props> {
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        if (!this.props._transcribing) {
+            return null;
+        }
+
+        return (
+            <Tooltip
+                content = 'This label indicates that there is a transcriber'
+                position = { 'left' }>
+                <CircularLabel
+                    className = 'recording-label'
+                    label = 'TR' />
```

Comment:
It'll be worthwhile to put the label and tooltip content through translation. Check out how other components use `t()` and strings in main.json.